### PR TITLE
Invite maintainers from community

### PR DESCRIPTION
**Pull request by smukherj1**
FYI @hicksjoseph 

Interest for this has already been expressed in #1743

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -1 +1 @@
-* @smukherj1 @davegay @eytankidron
+* @smukherj1
```
Comment (bergsieker) [inline CODEOWNERS:1]: Add Joe here?

File: README.md [hunk 2]
```diff
@@ -1,5 +1,15 @@
 # Bazel Container Image Rules
 
+## Are you interested in maintaining `rules_docker`?
```
Comment (bergsieker) [inline README.md:3]: s/maintaining/helping maintain/

**Review by bergsieker — APPROVED** [review]

File: CODEOWNERS (hunk 1 above)
Comment (smukherj1) [reply CODEOWNERS:1]: Done

**Pull request merged**